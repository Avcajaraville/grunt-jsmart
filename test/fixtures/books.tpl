{extends file='layout.tpl'}
{block name=body}
    <h1>{$books.greeting}</h1>

     {foreach $books.books as $i => $book}
         {include file='book.tpl' $book}
     {foreachelse}
         No books
     {/foreach}

     Total: {$book@total}
{/block}