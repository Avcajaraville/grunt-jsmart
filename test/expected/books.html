<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8">
	<title>Books Test</title>
</head>
<body>
	    <h1>Hi, there are some JScript books you may find interesting:</h1>

              <div style="background-color: cyan;">
 	[1] JAVASCRIPT THE DEFINITIVE GUIDE by David Flanagan
            Price: <span style="color:red">$31.18</span>
    </div>              <div style="background-color: yellow;">
 	[2] MURACH JAVASCRIPT AND DOM SCRIPTING by Ray Harris
            Price: <span style="color:red">$45.21</span>
    </div>              <div style="background-color: cyan;">
 	[3] HEAD FIRST JAVASCRIPT by Michael Morrison
            Price: <span style="color:red">$29.54</span>
    </div>     
     Total: 3
</body>
</html>